.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -91,4 +88,4 @@ An example changelog is attached to this PR for your convenience:
 -->
 :cl:
  * rscadd: Did stuff!
- * rscdel: did other stuff!
\ No newline at end of file
+ * rscdel: did other stuff!
```

Comment:
What's even changed on this line

Reply:
EOL

Reply:
How id that sneak in? Our EOLs should be standardized

Reply:
I guess the template is older than `.editorconfig`